opendata-python/tests/test_models.py

```diff
@@ -112,3 +117,23 @@ def test_store_locally(self, remote_athlete, local_storage):
             os.listdir(os.path.join(local_storage.strpath, settings.data_prefix))
         assert f'{{{remote_athlete.id}}}.json' in \
             os.listdir(os.path.join(local_storage.strpath, settings.metadata_prefix))
+
+
+class TestRemoteAthleteLocalStoringOptions:
```

Comment:
The `TestRemoteAthlete` is under the `@pytest.mark.aws_s3` decorator, while all my tests are running locally.

One way to fix it would be to decorate methods individually.

Reply:
These tests require access to S3 to pass too right? I think they should be under the same mark.

Reply:
ok, i see now that the Remote athletes is always fetching from the S3. So my arguments are incorrect and the test class should be indeed merged